File: Javascript/uri1010_CalculoSimples.js
var input = require("fs").readFileSync("stdin", "utf8");
var lines = input.split("\n");

var item1 = lines.shift().split(" ");
var item2 = lines.shift().split(" ");

var cod = item1.shift();
var num = item1.shift();
var vlr = item1.shift();

var cod2 = item2.shift();
var num2 = item2.shift();
var vlr2 = item2.shift();

function CalculoSimples(num, vlr, num2, vlr2){
    return ((num*vlr) + (num2*vlr2));
}

console.log("VALOR A PAGAR: R$ " + CalculoSimples(num, vlr, num2, vlr2).toFixed(2));

File: C/Iniciantes/Basicos/Uri1010_CalculoSimples.c
#include <stdio.h>
 
int main() {

 	int cod, num, cod2, num2;
	float valor, valor2, result;
    

    scanf("%i", &cod);
    scanf("%i", &num);
    scanf("%f", &valor);
    
    scanf("%i", &cod2);
    scanf("%i", &num2);
    scanf("%f", &valor2);
    
   result = ((num * valor) + (num2 * valor2));
   
    printf("VALOR A PAGAR: R$ %.2f\n", result );
 
    return 0;
}

